tests/templates/test_subroutines/test_qsvt.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the QSVT template and qsvt wrapper function.
+"""
+import pytest
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.templates.subroutines.qsvt import *
+
+
+class TestQSVT:
+    @pytest.mark.parametrize(
+        ("U_A", "lst_projectors", "wires", "expected_output"),
```

Comment:
How the `expected_output` values are computed?

Reply:
These are currently just the results of running the circuits.
I can replace these with something like \pi U_qsvt \pi to get better tests.